flipper-activerecord3dot2.gemspec

```diff
@@ -4,11 +4,14 @@ Gem::Specification.new do |gem|
   gem.authors       = ["Jesse Proudman", "Blake Gentry"]
   gem.email         = ["jproudman@bluebox.net", "blakesgentry@gmail.com"]
   gem.description   = %q{ActiveRecord 3.2 adapter for Flipper}
-  gem.summary       = %q{ActiveRecord 3.2 adapter for Flipper}
+  gem.summary       = <<-SUMMARY
+    A fork of `flipper-activerecord` which supports ActiveRecord 3.2 with minimal changes.
```

Comment:
Let's modify this to read:

A fork of `flipper-activerecord` which supports ActiveRecord 3.2 and ActiveRecord 4.1 with minimal changes.